data/json/monsters/zed_lieutenant.json

```diff
@@ -323,6 +318,37 @@
     "effect": [ { "math": [ "Lieutenants_active", "=", "1" ] }, { "math": [ "Shadow_Lurking", "=", "1" ] } ],
     "deactivate_condition": { "math": [ "Lieutenants_active", ">=", "1" ] }
   },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_LIEUTENANT_WARN_SHADOW",
+    "recurrence": [ "3 hours", "6 hours" ],
+    "global": true,
+    "run_for_npcs": false,
+    "condition": {
+      "and": [
+        { "days_since_cataclysm": 7 },
+        { "math": [ "days_since_cataclysm", "<", "10" ] },
+        { "one_in_chance": 3 },
```

Comment:
Two big problems I see here: First, this doesn't guarantee people ever get the warning (same issue as you faced) because it . Second, it's a 1 in 3 chance? So even if someone was running around, outside, at night, in fields and forests... they could still fail to get any warnings prior to it being spawnable just by sheer chance.